<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginLoaderDialog</class>
 <widget class="QDialog" name="PluginLoaderDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>load plugin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QToolBox" name="toolBox_Overview">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_local">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>977</width>
        <height>613</height>
       </rect>
      </property>
      <attribute name="label">
       <string>local</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="lbl_PluginType">
             <property name="text">
              <string>plugin type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_Name">
             <property name="text">
              <string>name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_Path">
             <property name="text">
              <string>path</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_pluginType">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_Name">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="lineEdit_Path">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_Path">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Description">
         <property name="title">
          <string>description</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextBrowser" name="textBrowser_description"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Cancel">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_Ok">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Ok</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>977</width>
        <height>613</height>
       </rect>
      </property>
      <attribute name="label">
       <string>network</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_Path</tabstop>
  <tabstop>toolButton_Path</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
